Test/Framework/Graphics/SpriteBatchTest.cs

```diff
@@ -296,7 +294,7 @@ public void DrawWithTexture()
         [Test]
         public void DrawWithCustomEffectAndTwoTextures()
         {
-            var customSpriteEffect = AssetTestUtility.CompileEffect(gd, "CustomSpriteBatchEffect.fx");
+            var customSpriteEffect = content.Load<Effect>(Paths.Effect("CustomSpriteBatchEffect"));
```

Comment:
@polsys and I also talked about this a little. How about we change these to AssetTestUtility.LoadEffecr and in there do

#if DIRECTX
    // compile and return effect
#else 
    // return a prebuilt effect
#endif